### PR TITLE
home page made responsive and added a burger menu

### PR DESCRIPTION
**Pull request by sarthakastic**
I have made the home page responsive.

### CODE REVIEW

**Comment by fykaa** [comment]
Lgtm, need two minor fixes-
1) please add a feature such that clicking anywhere on the screen closes the hamburger menu, since it's overlapping on different layouts
image



2) would you fix this scroll menu next to the search button as it wasn't present earlier - 
image

**Comment by sarthakastic** [comment]
ok

File: index.html [hunk 1]
```diff
@@ -110,14 +135,14 @@ <h3 class="subhead">
         <table style="width: 95%">
           <thead style="border: 2px solid black">
             <tr>
-              <th style="border: 1px solid rgb(173, 44, 44)">Project Name</th>
+              <th style="border: 1px solid rgb(173, 44, 44);  ">Project Name</th>
```
Comment (fykaa) [inline index.html:138]: the colon

Comment (fykaa) [inline index.html:138]: remove colon

**Comment by sarthakastic** [comment]
Now the navbar closes when we click anywhere. Please check it.

**Review by fykaa — APPROVED** [review]

**Pull request merged**